cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -965,6 +965,7 @@
    */
   public static final class Namespace {
     public static final String NAMESPACES_DIR = "namespaces.dir";
+    public static final String CREATE_DEFAULT_NAMESPACE = "create.default.namespace";
```

Comment:
Better call it `namespace.default.auto.creation`? We usually prefix config name, right?